### ISSUE TITLE
The link to API examples does not work

### ISSUE THREAD

**Issue by tisnik** [fabric8-analytics/fabric8-analytics-common#21]
In the README.md, there's link to API examples:
https://github.com/fabric8-analytics/examples

Unfortunately the link is not correct (probably caused by renaming repositories?)

**Comment by tisnik** [comment]
(please note that the link is used twice - in the API section and in the SonarQube section as well)

**Comment by fridex** [comment]
> (probably caused by renaming repositories?)

Yes, they were pointing to internal examples that shouldn't be relevant now. I think we can remove these links.
